### ISSUE TITLE
Regression in prefix handling

### ISSUE THREAD

**Issue by GavinMendelGleason** [terminusdb/terminusdb#1460]
Adding the following schema:

```
[
    {
        "@base": "http://purl.bdrc.io/resource/",
        "@schema": "http://purl.bdrc.io/ontology/core/",
        "@type": "@context",
        "adm": "http://purl.bdrc.io/ontology/admin/",
        "bda": "http://purl.bdrc.io/admindata/",
        "bdg": "http://purl.bdrc.io/graph/",
        "bdr": "http://purl.bdrc.io/resource/",
        "bdu": "http://purl.bdrc.io/resource-nc/user/",
        "bib": "http://id.loc.gov/ontologies/bibframe/",
        "cor": "http://purl.bdrc.io/ontology/core/",
        "owl": "http://www.w3.org/2002/07/owl#",
        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
        "skos": "http://www.w3.org/2004/02/skos/core#",
        "vcard": "http://www.w3.org/2006/vcard/ns#",
        "xsd": "http://www.w3.org/2001/XMLSchema#"
    },
    {
        "@id": "Title",
        "@type": "Class",
        "rdfs:label": {
            "@class": "rdf:langString",
            "@type": "Set"
        }
    },
    {
        "@id": "Note",
        "@type": "Class",
        "noteText": {
            "@class": "rdf:langString",
            "@type": "Set"
        }
    },
    {
        "@id": "Work",
        "@type": "Class",
        "bib:identifiedBy": "bdr:RefrKTsK",
        "isRoot": "xsd:boolean",
        "skos:altLabel": {
            "@class": "rdf:langString",
            "@type": "Set"
        },
        "skos:prefLabel": {
            "@class": "rdf:langString",
            "@type": "Set"
        },
        "workHasInstance": "Instance"
    },
    {
        "@id": "bdr:RefrKTsK",
        "@type": "Class",
        "rdf:value": "xsd:string"
    },
    {
        "@id": "Instance",
        "@type": "Class",
        "hasTitle": "Title",
        "instanceOf": "Work",
        "isRoot": "xsd:boolean",
        "note": "Note",
        "skos:prefLabel": "rdf:langString"
    },
    {
        "@id": "adm:ReleaseStatus",
        "@type": "Enum",
        "@value": [
            "StatusReleased",
            "StatusWithdrawn"
        ]
    },
    {
        "@id": "adm:AdminData",
        "@type": "Class",
        "adm:adminAbout": "Instance",
        "adm:facetIndex": "xsd:integer",
        "adm:gitPath": "xsd:string",
        "status": "adm:ReleaseStatus"
    }
]
```

Results in:

```API Error @type api:InsertDocumentErrorResponse api:message The key 'Title/http://www.w3.org/2000/01/rdf-schema#label/Set+rdf%3AlangString' is utilising an unknown prefix. api:status api:failure Code: 400 url: http://127.0.0.1:6363/api/document/admin/bdrc/local/branch/main```

**Comment by GavinMendelGleason** [comment]
This can not be reproduced via curl or CLI.

**Comment by matko** [comment]
If it can't be reproduced via curl or CLI, where can it be reproduced?

**Comment by GavinMendelGleason** [comment]
It was reported in the dashboard. I can reproduce it there as well. Could it be that alternative prefixes are used?

**Comment by matko** [comment]
Looking at the error, something is going wrong with the id `Title/http://www.w3.org/2000/01/rdf-schema#label/Set+rdf%3AlangString`.
This looks like the bug we saw last week where the `Title/http` part gets interpreted as a prefix, which then doesn't exist.
But I wouldn't expect this error to occur at schema insert time, but rather at document insert time.

**Comment by matko** [comment]
Ah, no, it is part of the schema of course. there's loads of sets of langstrings in there.
I'm pretty confused that this is not reproducible locally. Are we sure the dashboard has the latest versions deployed?

**Comment by matko** [comment]
I am actually able to reproduce locally.

**Comment by GavinMendelGleason** [comment]
Fixed by #1481
